moon/docker/rover/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ros:melodic
+
+SHELL ["/bin/bash", "-c"]
+
+RUN apt-get update && apt-get install -y python-pip python3-pip openssh-client vim less
+RUN python -m pip install pyzmq
+
+RUN /usr/bin/python3 -m pip install --user --no-cache-dir msgpack "opencv-python>3.4.2,<4" pyzmq
+
+COPY ./srcp2-competitors ./srcp2-competitors
```

Comment:
What is srcp2-competitors? It is not part of this PR.

Reply:
`srcp2-competitors` is private (unfortunately, i.e. not Open Source) repository and we had to sign NDA in order to get access to it. There is a chance that after the competition (maybe even after the qualification in August 2020) they (NASA) will make it publicly available.

It contains ROS messages specific for this competition and some tools needed to runt he simulation. Everything else in their proprietary docker.